source/_docs/configuration/packages.markdown

```diff
@@ -110,3 +110,10 @@ It is possible to [customize entities](/docs/configuration/customizing-devices/)
 homeassistant:
   customize:
 ```
+
+
+<div class='note warning'>
+
+If you are moving configuration to packages, auth_providers must stay within ‘configuration.yaml’. See [Issue 16441](https://github.com/home-assistant/core/issues/16441) and/ general documentation for [Authentication Providers](https://www.home-assistant.io/docs/authentication/providers/#configuring-auth-providers)
+
+This is because Home Assistant processes the `auth_provider` during the `core` section loading, which is earlier than the `packages` processing.
```

Comment:
```suggestion
This is because Home Assistant processes the `auth_provider` during the `core` section loading, which is done before the `packages` are processed.
```